src/main/ruby/post-boot/post-boot.rb

```diff
@@ -6,6 +6,10 @@
 # GNU General Public License version 2
 # GNU Lesser General Public License version 2.1
 
+if wd = Truffle::Boot.get_option('working_directory') and !wd.empty?
```

Comment:
`and` -> `&&` :-)

Reply:
But not here since I want `wd` to be the directory, not the result of `!wd.empty?`.
I could move the assignment out of the `if` though, it just feels more "scoped" this way.

Reply:
Ahh, right. See, this is why I hate that keyword.

Reply:
Fixed.